### ISSUE TITLE
install failed

### ISSUE THREAD

**Issue by ChrisDevinePimss** [WuglyakBolgoink/cordova-plugin-iroot#3]
cordova plugin add cordova-plugin-iroot --save
(node:6348) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): CordovaError: Cannot find plugin.xml for plugin "cordova-plugin-iroot". Please try adding it again.
(node:6348) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

also 

ionic cordova plugin add https://github.com/WuglyakBolgoink/cordova-plugin-iroot
> cordova plugin add https://github.com/WuglyakBolgoink/cordova-plugin-iroot --save
(node:12248) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): CordovaError: Failed to fetch plugin https://github.com/WuglyakBolgoink/cordova-plugin-iroot via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: cmd: Command failed with exit code 1 Error output:
npm ERR! code 128
npm ERR! Command failed: C:\Program Files\Git\cmd\git.EXE submodule update -q --init --recursive
npm ERR! fatal: 'submodule' appears to be a git command, but we were not
npm ERR! able to execute it. Maybe git-submodule is broken?
npm ERR!

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\chrisdevine\AppData\Roaming\npm-cache\_logs\2018-03-05T10_18_26_219Z-debug.log
(node:12248) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by WuglyakBolgoink** [comment]
@ChrisDevinePimss 

are you have already installed plugin before? or you try to install in new project?

**Comment by Chris-Devine** [comment]
@WuglyakBolgoink 

I installed into new project and it worked but prevented me from installing "cordova-plugin-network-information". So i had to uninstall it so i could install the rest of my plugins. 

I then tried to reinstall it once i had installed all the other plugins but got the errors above

**Comment by WuglyakBolgoink** [comment]
@Chris-Devine can you give me this information from
```bash
ionic info
```

I check this problem today night.

**Comment by ChrisDevinePimss** [comment]
PS C:\Users\chrisdevine\Desktop\agility-for-pimss\agility-for-pimss2> ionic info
[WARN] Error with .\www\lib\ionic\version.json file: FILE_NOT_FOUND, trying .\bower.json.

cli packages: (C:\Users\chrisdevine\AppData\Roaming\npm\node_modules)

    @ionic/cli-utils  : 1.19.1
    ionic (Ionic CLI) : 3.19.1

global packages:

    cordova (Cordova CLI) : 8.0.0
    Gulp CLI              : CLI version 2.0.1 Local version 3.9.1

local packages:

    Cordova Platforms : none
    Ionic Framework   : ionic1 1.3.4

System:

    Node : v8.9.4
    npm  : 5.6.0
    OS   : Windows 10

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : pro

**Comment by WuglyakBolgoink** [comment]
@ChrisDevinePimss / @Chris-Devine with which platforms will you work? Android or iOS?

**Comment by ChrisDevinePimss** [comment]
yes thats the plan @WuglyakBolgoink

**Comment by Medtrack** [comment]
I can confirm this bug on Android.
Cordova version 8.0.0 and cordova-android@~7.0.0
Just create a script or run the following commands:

```
#!/usr/bin/env bash
currentDir=$(pwd)
cordova create hello com.example.hello HelloWorld
cd hello
cordova plugin add cordova-plugin-iroot --save
cordova platform add android
cd $currentDir
```

**Comment by sujit77** [comment]
@Medtrack  I have added working solution for this issue [#1](issuecomment-372641174). Try removing `.git` folder from package folder under `node_modules`.

If that doesn't works, delete `node_modules` and `plugins` folders and `package-lock.json` file and reinstall plugins using `npm i` in your project folder.
Note - You have to remove `.git` folder(as mentioned above) before building android apk file.

**Comment by WuglyakBolgoink** [comment]
@Chris-Devine @SujitSingh  please check with new v0.7.0. `npm publish` was without errors.

**Comment by SujitSingh** [comment]
The error reported in this issue is related to the extra `.git` folder which has been resolved in #6
